swan/scripts/before-notebook.d/02_environment.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# Author: Danilo Piparo, Enric Tejedor, Pedro Maximino, Diogo Castro 2023
+# Copyright CERN
+# Here the environment for the notebook server is prepared. Many of the commands are 
+# launched as regular user as it's this entity which is able to access 
+# eos and not the super user.
+
+# Set environment for the Jupyter process
+LOCAL_HOME=/home/$NB_USER
+export IPYTHONDIR=$LOCAL_HOME/.ipython
+export JPY_DIR=$LOCAL_HOME/.jupyter
+
+# JUPYTER_CONFIG_DIR is set to point to the local user home (/home/$USER)
+# instead of the EOS path
+export JUPYTER_CONFIG_DIR=$JPY_DIR
+export JUPYTER_PATH=$LOCAL_HOME/.local/share/jupyter
+
+# Set other environment variables
+export KERNEL_DIR=$JUPYTER_PATH/kernels
+export SWAN_ENV_FILE=$LOCAL_HOME/.bash_profile
+export PROFILEPATH=$IPYTHONDIR/profile_default
+
+# Create missing directories
+mkdir -p $IPYTHONDIR $PROFILEPATH
+
+# Setup the LCG View on CVMFS
+_log "Setting up environment from CVMFS"
+export LCG_VIEW=$ROOT_LCG_VIEW_PATH/$ROOT_LCG_VIEW_NAME/$ROOT_LCG_VIEW_PLATFORM
+
+# symlink $LCG_VIEW/share/jupyter/nbextensions for the notebook extensions
+ln -s $LCG_VIEW/share/jupyter/nbextensions $JUPYTER_PATH
+#Creating a ROOT_DATA_DIR variable
+export ROOT_DATA_DIR=$(readlink $LCG_VIEW/bin/root | sed -e 's/\/bin\/root//g')
```

Comment:
We should try to remove this and add it to the list of things to try:
- 1 (see comment before)
- 2 (see comment before)
- 3: Test ROOT without ROOT_DATA_DIR